app/subscriber/src/features/my-folders/styled/FolderLanding.tsx

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+import { Row } from 'tno-core';
+
+export const FolderLanding = styled(Row)<{ split?: boolean }>`
```

Comment:
Split is true from the FolderLanding component when an action is defined 
i.e) `/folders/configure/5` 
In this case configure is the action and will cause the screen to split. Follows the format `/action/id`